modules/tinymce/src/core/main/ts/textpatterns/core/InlinePattern.ts

```diff
@@ -167,18 +166,6 @@ const findPatternsRec = (
     rng.setEnd(node, offset);
     const text = rng.toString();
 
-    // TINY-8781: TODO: text_patterns should announce their changes for accessibility
-    const extraPatterns = patternSet.dynamicPatternsLookup({
-      text,
```

Comment:
Hmm... I hadn't noticed this when discussing last week sorry Ben, but the new implementation returns the whole block not from the start of the trigger character back to the first element. Is that going to be an issue?

I think it is and the inline key logic needs to do something like this instead of the whole block, as if we trigger something in the middle of a paragraph then we really only want to deal with where the text pattern is starting to match, not the whole block.

Reply:
It is an issue. So I changed the code to only pass the text from the beginning to the cursor location into `dynamicPatternsLookup` when matching inline patterns. Also added a test where matching is triggered in the middle of the paragraph. 

Not sure whether passing extra info such as matching mode (inline or block) to the `dynamicPatternsLookup` would be helpful. Maybe leave it for now until there is a clear use case for it